### PR TITLE
Implemented 'edit apartment' function

### PR DESCRIPTION
**Pull request by EllieChen-Git**

### CODE REVIEW

File: controllers/image_controller.js [hunk 1]
```diff
@@ -73,17 +76,15 @@ async function destroy(req, res) {
...
-  // res.json(updatedImage);
-  console.log("Update");
+async function update(req, res, next) {
+  console.log(req.body); //Keep it here to see info on console
+
+  await ImageModel.findByIdAndUpdate(req.params.id, {
+    lot: req.body.lot,
+    unitNumber: req.body.unitNumber,
+    productDescription: req.body.productDescription,
+    s3key: req.body.selectedFile // Currently, I do it with a cheeky way. When updating an apartment, the s3key in our database will be updated. But the file is not on AWS S3 hahahahah
```
Comment (EllieChen-Git) [inline controllers/image_controller.js:86]: See my comment in code & here. 
Q1: Still need to figure out how to upload files to s3 if users decide to upload a new image when editing apartments.
Q2: If users don't upload any image when editing apartment, the app will break. So currently I make it mandatory on frontend (i.e. when users want to edit apartments, they must pick a new image haha).
Need to discuss this with you tmr :)

File: routes/images.js [hunk 2]
```diff
@@ -40,21 +41,21 @@ router.delete(
...
-        .items(
-          Joi.object({
-            id: Joi.string().required(),
-            s3key: Joi.string().required()
-          })
-        )
-        .min(1)
-    }
-  }),
+// Uppdate an Image: If I turn celebrate on, the edit function won't work. I need to learn how to use celebrate :(
```
Comment (EllieChen-Git) [inline routes/images.js:44]: I don't know how to use celebrate when editing image.
Will ask you tmr.

File: controllers/image_controller.js (hunk 1 above)
Comment (INRokh) [reply controllers/image_controller.js:86]: ok, thanks

File: routes/images.js (hunk 2 above)
Comment (INRokh) [reply routes/images.js:44]: ok

**Review by INRokh — APPROVED** [review]

**Pull request merged**